src/backend/.docker/Dockerfile.local

```diff
@@ -1,5 +1,27 @@
 FROM node:18.9-alpine
 
+RUN apk update
+
+# Installs latest Chromium (100) package.
+RUN apk add --no-cache \
+      chromium \
+      nss \
+      freetype \
+      harfbuzz \
+      ca-certificates \
+      ttf-freefont \
+      nodejs \
+      yarn
+
+# Puppeteer v13.5.0 works with Chromium 100.
+# RUN yarn add puppeteer@13.5.0
```

Comment:
I see yarn in the local dockerfile, so it makes sense to have it here. Can we remove it from the actual dockerfile?

Reply:
this line is commented and can be removed actually. We are using npm and shall stay consistent to it. I will get rid of it here and in the Dockerfile too.